docs/reference/api/docker_remote_api_v1.20.md

```diff
@@ -140,9 +140,14 @@ Create a container
                    "com.example.license": "GPL",
                    "com.example.version": "1.0"
            },
-           "Volumes": {
-                   "/tmp": {}
-           },
+           "Mounts": [
+             {
+               "Source": "/data",
+               "Destination": "/data",
+               "Relabel": "ro,Z",
```

Comment:
my point was mainly that (iiuc), `ro`/`rw` is not related to relabeling, so found that a bit odd.